### PR TITLE
Add cell copy paste

### PR DESCRIPTION
**Pull request by captainsafia**
For some weird reason, it won't detect `Cmd + C` properly.  `handleKeyDown` seems to be configured properly to detect it but I might be missing something obvious.

Also, this is set up in way to allow multi-cell copy paste in the future.

cell-copy-paste

### CODE REVIEW

**Comment by rgbkrk** [comment]
YaY!

File: src/notebook/components/cell/cell.js [hunk 1]
```diff
@@ -74,6 +85,24 @@ class Cell extends React.Component {
...
+    if (this.state.ctrlDown) {
+      if (event.keyCode === 86) {
+        this.setState({ ctrlDown: false });
+        this.pasteCell();
+      } else if (event.keyCode === 67) {
+        this.setState({ ctrlDown: false });
+        this.copyCell();
+      }
+    }
+  }
```
Comment (rgbkrk) [inline src/notebook/components/cell/cell.js:104]: I'd look to the markdown test to see how to test these two keyboard driven functions.

**Comment by rgbkrk** [comment]
I'd check that the copy action was dispatch (using `sinon` spies or otherwise). Checking state will work for now if it at least gets the coverage though.

**Comment by captainsafia** [comment]
> I'd check that the copy action was dispatch (using sinon spies or otherwise). Checking state will work for now if it at least gets the coverage though.

I'm gonna do this know just so I don't get in the habit of writing bad tests. Gimme 2 seconds.

**Comment by captainsafia** [comment]
:shipit: !

File: test/renderer/components/cell/cell-spec.js [hunk 2]
```diff
@@ -32,4 +34,39 @@ describe('Cell', () => {
...
+      <Cell cell={commutable.emptyCodeCell} {...sharedProps}/>,
+      { context: { store: dummyStore() } }
+    );
+
+    const spy = sinon.spy(cell.instance(), "pasteCell"); 
+
+    cell.simulate('keydown', { key: 'Ctrl', ctrlKey: true});
+    cell.simulate('keyup', { keyCode: 86 });
+    expect(cell.state('ctrlDown')).to.be.false;
+    expect(spy.called).to.be.true;
```
Comment (rgbkrk) [inline test/renderer/components/cell/cell-spec.js:70]: 👍

**Comment by rgbkrk** [comment]
Well this is awesome.

**Comment by captainsafia** [comment]
Me so happy.

**Pull request merged**